Makefile

```diff
@@ -65,6 +67,10 @@ help:
 	@echo '  docscheck     Run `mkdocs build --strict`'
 	@echo '  lint          Run `ruff check`'
 	@echo '  formatcheck   Run `ruff format --check` (check only, do not modify files)'
+	@echo ''
+	@echo 'Other useful commands:'
+	@echo '  make install UV_ARGS=--no-dev   Minimal Pixeltable installation (no dev packages)'
```

Comment:
nit: we list other commands without `make ` and should probably keep it consistent

Reply:
Right, this was an intentional choice - the others are lists of targets, whereas these are fully-baked command lines, so I thought it made sense to include `make` for clarity. But I don't feel strongly about it.